backend/src/services/real_events_service.py

```
@@ -277,11 +277,34 @@ async def _fetch_perplexity_events(self, destination: str, start_date: str, end_
+
+OUTPUT:"""
                         
                         try:
+                            try:
+                                cleaned_content = await llm_parser._parse_with_openai(citation_cleaning_prompt)
+                                if isinstance(cleaned_content, str):
+                                    content = cleaned_content
+                                elif isinstance(cleaned_content, dict) and 'events' in cleaned_content:
+                                    events = cleaned_content['events']
+                                    return events[:10] if len(events) > 10 else events
+                            except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)